src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -8,13 +8,17 @@ export type ColumnValueType =
   | 'boolean'
   | 'never'; // Matches to nothing
 
+type AGGREGATION_KEYS = 'count' | 'count_unique' | 'min' | 'max' | 'sum' | 'avg';
```

Comment:
Could this type be generated using `keyof AGGREGATIONS` after `AGGREGATIONS` is defined? Or use an untyped map to generate the typed one? Right now adding a new function requires updates in multiple places.

Reply:
@markstory the alternative I can think of is this https://github.com/getsentry/sentry/pull/15819/commits/3bc8fa248de829291a20c9e8ac8cc9d42756e8b0